src/views/Wrap/Wrap.tsx

```diff
@@ -217,155 +217,155 @@ const Wrap: React.FC = () => {
-          <Paper className={`ohm-card`}>
-            <Grid container direction="column" spacing={2}>
-              <Grid item>
-                <div className="card-header">
-                  <Typography variant="h5">Wrap / Unwrap</Typography>
-                  <Link
-                    className="migrate-sohm-button"
-                    style={{ textDecoration: "none" }}
-                    href={
-                      assetTo === "wsOHM"
-                        ? "https://docs.olympusdao.finance/main/contracts/tokens#wsohm"
-                        : "https://docs.olympusdao.finance/main/contracts/tokens#gohm"
-                    }
-                    aria-label="wsohm-wut"
-                    target="_blank"
-                  >
-                    <Typography>gOHM</Typography>{" "}
-                    <SvgIcon component={ArrowUp} color="primary" style={{ marginLeft: "5px", width: ".8em" }} />
-                  </Link>
-                </div>
-              </Grid>
-              <Grid item>
-                <MetricCollection>
-                  <Metric
-                    label={`sOHM ${t`Price`}`}
-                    metric={formatCurrency(sOhmPrice, 2)}
-                    isLoading={sOhmPrice ? false : true}
-                  />
-                  <Metric
-                    label={t`Current Index`}
-                    metric={trim(currentIndex, 1)}
-                    isLoading={currentIndex ? false : true}
-                  />
-                  <Metric
-                    label={`gOHM ${t`Price`}`}
-                    metric={formatCurrency(gOhmPrice, 2)}
-                    isLoading={gOhmPrice ? false : true}
-                    tooltip={`gOHM = sOHM * index\n\nThe price of gOHM is equal to the price of sOHM multiplied by the current index`}
-                  />
-                </MetricCollection>
-              </Grid>
-              <div className="staking-area">
-                {!address ? (
-                  <div className="stake-wallet-notification">
-                    <div className="wallet-menu" id="wallet-menu">
-                      {modalButton}
-                    </div>
-                    <Typography variant="h6">Connect your wallet</Typography>
+          <Paper
+            className="ohm-card"
+            headerText="Wrap / Unwrap"
+            topRight={
+              <Link
+                className="migrate-sohm-button"
+                style={{ textDecoration: "none" }}
+                href={
+                  assetTo === "wsOHM"
+                    ? "https://docs.olympusdao.finance/main/contracts/tokens#wsohm"
+                    : "https://docs.olympusdao.finance/main/contracts/tokens#gohm"
+                }
+                aria-label="wsohm-wut"
+                target="_blank"
+              >
+                <Typography>gOHM</Typography>{" "}
+                <SvgIcon component={ArrowUp} color="primary" style={{ marginLeft: "5px", width: ".8em" }} />
```

Comment:
You can import Icon from component-library and use that instead.
```suggestion
               <Icon style={{ marginLeft: "5px" }} name="arrow-up" />
```

Reply:
fixed